spec/requests/api/v2/conferences_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe "/api/v2/conferences" do
+  describe "get conferences" do
+    context "for JSON" do
+      it "will render all required keys for v2" do
+        create(:conference, :ruby)
+
+        get "/api/v2/conferences", { tags: 'ruby' }, token_auth
+
+        expect(json.count).to eq(1)
+        expect(json.first.keys).to eq(
+          [
+            "id",
+            "name",
+            "location",
+            "twitter_username",
+            "start_date",
+            "end_date",
+            "website",
+            "when",
+            "logos"
+          ]
+        )
+      end
+
+      it "will render conferences tagged with given tags" do
+        create(:conference, :ruby)
+        create(:conference, :javascript)
+        design_conference = create(:conference, :design)
+
+        get "/api/v2/conferences", { tags: 'ruby,javascript' }, token_auth
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.